styles/globals.css

```diff
@@ -6,3 +6,11 @@
 body {
   @apply font-rubik;
 }
+.btn-primary {
+  @apply bg-primary text-white	 border-2 rounded-lg border-primary py-1 px-2.5 filter hover:brightness-95 transition duration-700 ease-in-out;
```

Comment:
I think the duration is too much,`duration-200` or `duration-100` would be better imo.

Reply:
it was suggested by lana